src/Pact/Types/KeySet.hs

```diff
@@ -98,7 +111,8 @@ instance Pretty KeySet where
 
 instance SizeOf KeySet where
   sizeOf (KeySet pkArr ksPred) =
-    (constructorCost 2) + (sizeOf pkArr) + (sizeOf ksPred)
+    -- TODO: check this ns sizeOf later
+    constructorCost 2 + sizeOf pkArr + sizeOf ksPred
```

Comment:
This will require forking unfortunately.

Reply:
this is the same just removed the unnecessary parens. Looks like it was left over from the refactor - reverting the line